server/WebAPI/Services/ScheduledPlaceToEatService.cs

```diff
@@ -57,25 +59,34 @@ public int CreateNewEating(InputScheduledPlaceToEatForCreateDTO scheduledPlaceTo
       }
     }
 
-    public IEnumerable<OutputScheduledPlaceToEatDTO> GetEatingByUserId(int UserId)
+    public IPagedResponse<OutputScheduledPlaceToEatDTO> GetEatingByUserId(int UserId, int page, int pageSize)
     {
       if(_userService.GetUser(UserId) != null)
       {
         var eatings = _scheduledPlaceToEatRepository.GetScheduledPlaceToEatByUserId(UserId);
+        var TotalCount = eatings.Count();
+        eatings = eatings.Skip(page * pageSize).Take(pageSize);
         List<OutputScheduledPlaceToEatDTO> result = new List<OutputScheduledPlaceToEatDTO>();
         foreach(var eating in eatings)
         {
           result.Add(AddFileNames(eating));
         }
-        return result;
+        return new IPagedResponse<OutputScheduledPlaceToEatDTO>()
+        {
+          Data = result,
+          PageNumber = page,
+          PageSize = pageSize,
+          PageCount = TotalCount / pageSize,
```

Comment:
You can use Math.Ceiling() here to avoid +1 on front

Reply:
ok